app/services/reports/querier.rb

```diff
@@ -2,38 +2,53 @@ module Reports
 
   class Querier
 
+    # This seems to be ideal for NU/Oracle, but we can consider tweaking for MySQL
```

Comment:
Have we tested the batching against MySQL datasets? Just want to make sure this doesn't slow things down for them.

Reply:
I have not yet. That is something I do want to test out before we release this.